Docs/update readme for podman (#45)
eclipse/chariott: README.md

## Getting started

> ### Important Note

Reviewer: This section should be called **Requirements** and go before **Getting Started**.

Author: Done in https://github.com/eclipse/chariott/pull/45/commits/771c33ec91fa9fd935cf97e32c61f6eb82d8e1df

The current source is developed and tested under WSL2 / Linux running Ubuntu 20.04
on AMD64 architecture. It is not tested against any other configurations. You

> might experience missing support for other platforms, please feel free to

Reviewer: ```suggestion
might experience missing support for other platforms, but please feel free to
```

contribute to close the gaps.

### Devcontainer

For development and running the examples, we recommend using the

> [Devcontainer](https://code.visualstudio.com/docs/remote/containers) template

Reviewer: ```suggestion
[dev container](https://code.visualstudio.com/docs/remote/containers) template
```

cargo test --workspace
```

### Using Podman instead of Docker

> If you want to use podman you have to enable podman in Visual Studio Code like
> described [here](https://code.visualstudio.com/remote/advancedcontainers/docker-options#_podman)
> and update the `.devcontainer/devcontainer.json` file with the following additions:

Reviewer: ```suggestion
If you want to use Podman you have to [enable Podman in Visual Studio
Code][vscode-podman] and update the `.devcontainer/devcontainer.json` file
with the following additions:

  [vscode-podman]: https://code.visualstudio.com/remote/advancedcontainers/docker-options#_podman
```

Suggestions:

- Capitalize “Podman”
- Move link out of main text.

> ```Dockerfile

Reviewer: But this is snippets of a Dockerfile and therefore probably not something that a syntax highlighter will be happy with so may be remove this?

Author: Originally, I wanted to place an alternative Dockerfile into the `.devcontainer/` folder. but decided to give at least the corresponding hints on what to change if someone wants to use it with Podman. The snippet should just help getting the right changes, maybe splitting them into separate chunks?

Author: Included your suggestion using jsonc as code fence block syntax

cargo test --workspace
```

### Using Podman instead of Docker

If you want to use Podman you have to [enable Podman in Visual Studio
Code][vscode-podman] and update the `.devcontainer/devcontainer.json` file
with the following additions:

  [vscode-podman]: https://code.visualstudio.com/remote/advancedcontainers/docker-options#_podman

> ```Dockerfile
>   "runArgs": [
>     "--cap-add=SYS_PTRACE",
>     "--security-opt",
>     "seccomp=unconfined",
>     "--userns=keep-id"
>   ],
> 
>   ...
> 
>   "workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind,Z",
>   "workspaceFolder": "/workspace",
>   "containerUser": "vscode"
> ```

Reviewer: > The snippet should just help getting the right changes, maybe splitting them into separate chunks?

How about just putting it in [JSONC](https://changelog.com/news/jsonc-is-a-superset-of-json-which-supports-comments-6LwR) format (and which is what this is instead of `Dockerfile`) where you use `/* ... */` to mark _other stuff_?

```suggestion
```jsonc
{
  /* ... */
  "runArgs": [
    "--cap-add=SYS_PTRACE",
    "--security-opt",
    "seccomp=unconfined",
    "--userns=keep-id"
  ],

  /* ... */
    
  "workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind,Z",
  "workspaceFolder": "/workspace",
  "containerUser": "vscode",
  /* ... */
}
```
```
